networkx/algorithms/community/tests/test_local.py

```diff
@@ -0,0 +1,67 @@
+import networkx as nx
+
+
+def test_clauset_karate_club():
+    G = nx.karate_club_graph()
+
+    community = nx.community.clauset_greedy_source_expansion(G, source=16)
+
+    expected = {0, 4, 5, 6, 10, 16}
+
+    assert community == expected
+
+
+def test_clauset_cutoff():
+    G = nx.karate_club_graph()
+
+    community = nx.community.clauset_greedy_source_expansion(G, source=16, cutoff=3)
+
+    assert len(community) == 3
```

Comment:
The result here should be deterministic given this set of inputs, right? If that's the case then I'd vote make the test more stringent by checking the community as a whole instead of only it's size:

```suggestion
    community = nx.community.clauset_greedy_source_expansion(G, source=16, cutoff=3)

    assert community == {5, 6, 16}
```